<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MacBook-Pro.local" failures="1" tests="5" name="com.qa.opencart.tests.LoginPageTest" time="14.638" errors="0" timestamp="2025-10-03T23:36:24 IST" skipped="0">
  <testcase classname="com.qa.opencart.tests.LoginPageTest" name="forgotPswLinkExist" time="0.917"/>
  <system-out/>
  <testcase classname="com.qa.opencart.tests.LoginPageTest" name="forgotpsw" time="0.003"/>
  <system-out/>
  <testcase classname="com.qa.opencart.tests.LoginPageTest" name="loginPageTitleTest" time="10.502">
    <failure message="expected [Account Login11] but found [null]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Account Login11] but found [null]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.qa.opencart.tests.LoginPageTest.loginPageTitleTest(LoginPageTest.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- loginPageTitleTest -->
  <system-out/>
  <testcase classname="com.qa.opencart.tests.LoginPageTest" name="loginPageURLTest" time="0.011"/>
  <system-out/>
  <testcase classname="com.qa.opencart.tests.LoginPageTest" name="dologinTest" time="3.205"/>
  <system-out/>
</testsuite> <!-- com.qa.opencart.tests.LoginPageTest -->
